Makefile

```
@@ -14,7 +14,7 @@ test-docker:
 	docker run twilio/twilio-go go test -race ./...
 
 cluster-test:
-	go test -race --tags=cluster
+	go test -race --tags=cluster ./...
```

```suggestion
	go test -trimpath -race --tags=cluster ./...
```

This flag can help with build caching.